src/controller/CommissioningDelegate.cpp

```diff
@@ -137,6 +137,10 @@ const char * StageToString(CommissioningStage stage)
         return "FindOperational";
         break;
 
+    case kIcdIdentification:
```

Comment:
We should probably use ICD, not Icd, in names...

Reply:
this line no longer exists